packages/bot/src/components/run-panel.jsx

```diff
@@ -46,19 +49,57 @@ const drawerFooter = (props) => {
     );
 };
 
-const RunPanel = () => {
-    const content = drawerContent();
-    const footer = drawerFooter(onRunBotClick);
+// Temperarily change to state component to test animation
+class RunPanel extends React.PureComponent {
+    constructor() {
+        super();
+        this.state = {
+            contract_status: 0,
```

Comment:
@sam-binary please remove states and use observable. you can define it in run-panel-store

Reply:
this is just temporarily, so show the animation. Once integrated with Aaron part, all these will be removed